src/gmp.c

```diff
@@ -17223,7 +17223,7 @@ handle_get_scanners (gmp_parser_t *gmp_parser, GError **error)
           if (!agent_control_get_details_from_iterator (&scanners, &desc, &params))
             {
               SENDF_TO_CLIENT_OR_FAIL
-               ("<info><scanner><name>agent-controller</name><version>0.1</version>"
+               ("<info><scanner><name>agent-controller-sensor</name><version>0.1</version>"
```

Comment:
I think this should remain as "agent-controller" because it is the actual scanner is the same but OSP sensors are also not handled, so maybe we should clarify this first.